controls/def.cf

```diff
@@ -633,6 +633,7 @@ bundle common inventory_control
     !freebsd::
 
       "dmidecoder" string => "/usr/sbin/dmidecode";
+      "proc_device_tree" string => "/proc/device-tree";
```

Comment:
I don't love this being here, feels like it should be in inventory/linux.cf, or push the have_proc_device_tree class here and reference in both any.cf and linux.cf.